mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -28,6 +28,7 @@
 import java.util.Map;
```

Comment:
The commit name do not specify what you really do : **Removing annotations when you remove mailbox**

I would like to see integration tests on this : 

1/ 
 - Create a mailbox
 - Add annotations to it
 - Delete mailbox
 - (Wait)
 - Check annotations. Should complain mbx do not exist

2/ 

 - Create a mailbox
 - Add annotations to it
 - Delete mailbox
 - (Wait)
 - Create mailbox again
 - Check annotations. Should be empty